### PR TITLE
AAQ-292: Docs add blog

### PR DESCRIPTION
**Pull request by sidravi1**
Reviewer: @Tanmay-97 

Estimate: 15 mins

---

## Ticket

Fixes: [AAQ-292](https://idinsight.atlassian.net/browse/AAQ-292)

### Goal

I want to share with stakeholder what new features we are rolling out and what we have been working on.
A blog in docs seem like a good way to do it.

### Changes

Setup blog frameworks

## How has this been tested?

Run docs locally:

```
mkdocs serve -a localhost:8001
```

## Checklist

Fill with `x` for completed. Delete any lines that are not relevant

- [x] My code follows the style guidelines of this project
- [x] I have reviewed my own code to ensure good quality
- [x] I have tested the functionality of my code to ensure it works as intended
- [x] I have resolved merge conflicts
- [x] I have updated the automated tests (if applicable)
- [x] I have updated the requirements (if applicable)
- [x] I have updated the README file (if applicable)
- [x] I have updated affected documentation (if applicable)

[AAQ-292]: https://idinsight.atlassian.net/browse/AAQ-292?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -71,6 +71,9 @@ nav:
       - Guidelines: develop/contributing.md
       - Setup: develop/setup.md
       - Testing: develop/testing.md
+  - Latest Updates:
```
Comment (Tanmay-97) [inline mkdocs.yml:74]: Suggesting this to be the 2nd tab after "What is AAQ?" and title it `⚠️ Latest Updates` or `📢 Latest Updates` instead.
Other emoji option - 🚨❗

File: mkdocs.yml [hunk 2]
```diff
@@ -71,6 +71,9 @@ nav:
       - Guidelines: develop/contributing.md
```
Comment (Tanmay-97) [inline mkdocs.yml:71]: Suggested icons:
```
🏠 Welcome to AAQ
⚠️ Latest Updates
⚙️ Core Components
➕ Other Components
🌀 Deploying
💝 Contributing
```

Comment (Tanmay-97) [reply mkdocs.yml:71]: image

**Review by Tanmay-97 — APPROVED** [review]
Nice! Cool blog! Suggested some beautification.

File: mkdocs.yml (hunk 2 above)
Comment (sidravi1) [reply mkdocs.yml:71]: Sounds good! Done

File: mkdocs.yml (hunk 1 above)
Comment (sidravi1) [reply mkdocs.yml:74]: Great! Moved it.

**Pull request merged**